lib/at_ex.ex

```diff
@@ -169,11 +169,32 @@ defmodule AtEx do
   """
   defdelegate b2c_checkout(map), to: Payments.Mobile.B2c
 
+  @doc """
```

Comment:
This file seems is getting bigger, What is your opinion about breaking this further later, so we can have something like `AtEx.Payment` to handle all payment endpoints then `AtEx.Voice` for voice endpoints and so forth.

Reply:
@kamalogudah yeah, this is a good suggestion, maybe we can do this after everything id done, maybe  a last PR before we have our version 1

Reply:
Why I say we it later, some of the Voice and Payments are already on that file, so it can be a refactor, but there's no harm if we that in this PR too.